docs/pages/eas/metadata/schema.mdx

```diff
@@ -9,7 +9,7 @@ import { MetadataTable, MetadataSubcategories } from '~/components/plugins/EasMe
 import { Collapsible } from '~/ui/components/Collapsible';
 import { markdownComponents as MD } from '~/ui/components/Markdown';
 
-> **warning** EAS Metadata is in preview and subject to breaking changes.
+> **important** **EAS Metadata** is in preview and subject to breaking changes.
```

Comment:
Can you please apply this important callout to EAS Hosting pages too? Such as https://docs.expo.dev/eas/hosting/introduction/